src/create-context.js

```diff
@@ -34,11 +29,11 @@ export const createContext = (defaultValue, contextId) => {
 				ctx = {};
 				ctx[contextId] = this;
 				this.getChildContext = () => ctx;
-				providers.add(this._internal);
+				providers.add(this);
 			}
 			// re-render subscribers in response to value change
 			else if (props.value !== this._prev) {
-				this._subs.forEach(enqueueRender);
+				this._subs.forEach(i => i.render());
```

Comment:
```suggestion
				this._subs.forEach(i => { i.render() });
```

Saves some bytes

Reply:
ha! can't believe I missed that.

Reply:
@JoviDeCroock would you believe that actually adds 3b?! gzip

Reply:
Maybe it comes from the `return` keyword added at build time 🤷